### DISCUSSION TITLE
TEMP CHECK: Q4 2024 | Management, Maintenance, and Development Proposal

### DISCUSSION THREAD

**Discussion by nblogist** [ringdao-community-guild/discussion#186] [Proposals Temperature Check]
Due to bear market - the rates have been adjusted temporarily refer to #183 

Darwinia Community DAO
Prepared by: Furqan A.
---
## Introduction
This Q4 proposal builds upon the accomplishments of Q3 and outlines a continued focus on Marketing, Branding, and Website Maintenance. I am very happy to share that we decided on a uniform marketing plan and are working on combined efforts to improve Darwinia's exposure - I am also happy to share that thanks to Saso's connection we are now represented at [Cosmoverse 2024](https://cosmoverse.org/speakers). My team at DotMatrix will maintain the same roles and responsibilities, ensuring consistent progress across all initiatives.

## Summary of Accomplishments in Q3
Q3 Proposal: [Link](https://github.com/orgs/dcdao/discussions/166)
Creating Developer Materials - Video Series: [Link](https://github.com/orgs/dcdao/discussions/167)
Videos published on Darwinia's Youtube: [Link](https://www.youtube.com/@darwinianetwork/videos)
July 2024 Report: [Link](https://github.com/orgs/dcdao/discussions/172)
August 2024 Report: [Link](https://github.com/orgs/dcdao/discussions/176)
Marketing Action Plan 2024-2025: [Link](https://www.tally.xyz/gov/dcdao/proposal/29816049634754307523658710595374865934730721906103021309486532985405898633548)
September 2024 Report: [Link](https://github.com/orgs/dcdao/discussions/182)
Throughout Q3, I focused on developing and executing marketing strategies, refining the Darwinia branding, and coordinating efforts within the team. Additionally, we built, maintained and optimized the Darwinia, RingDAO and Paralink websites, ensuring a bug-free, updated platform.

## Proposed Roles and Responsibilities for Q4
1. Marketing and Branding Management

I will continue to lead marketing and branding efforts, ensuring a consistent strategy that builds on Q3 accomplishments. Responsibilities building on the previous ones will include:

Continue developing and executing high-level marketing strategies to promote Darwinia's projects and initiatives.
Managing branding efforts to ensure consistent and effective messaging across all platforms.
Organizing and representing Darwinia at real-life events to increase community engagement and visibility.
Collaborating with the community to gather feedback and adjust marketing strategies accordingly.
Coordinating with external partners and influencers to expand Darwinia's reach and impact.
Overseeing the strategic direction of marketing campaigns and ensuring alignment with overall organizational goals.

2. Darwinia Website Maintenance

The ongoing website components creation and maintenance will continue as before, ensuring the Darwinia.Network and related platforms remain functional, bug-free, and consistently updated with new features when needed.

Amount proposed for these roles: $3,500/month

Please feel free to reach out with any questions or suggestions. Your feedback is crucial to our DCDAO's success.

**Comment by SasoLithops** [comment]
Your contribution so far to start reaching out to projects at real-life events and work in marketing and branding is a step in right direction imho. To continue in Q4 and to visit at least one major event more I will vote AYE.

**Reply by nblogist** [reply]
Saso - I would love to - but we need budget for that - and seeing the last proposal - it seems difficult - even if i use my connections to get us discounts etc.
What would you recommend - I know we need to do more events but - my hands are tied.

**Comment by nblogist** [comment]
Added to Tally for voting!
https://www.tally.xyz/gov/dcdao/proposal/44950363365822688680355567822968680664271449679282481069711552299087627792091
